### ISSUE TITLE
Scan error on FSL2-RU sensor change

### ISSUE THREAD

**Issue by AgushkaNightscout** [j-kaltes/Juggluco#154]
I have [same#41](https://github.com/j-kaltes/Juggluco/issues/41) issue with L2 RU version sensor. My dia-phone (Cubot Poket, Android 10) can't intercept already activated (by Realme 10) with Abbot's original app L2-RU sensor with NFC (error 18), although L3 could activate. But another phone (Realme 10, Android 13) can (no error)! I activate sensor with Realme and then transfer the sensor to Cubot using Juggluco's "Get Glucose values on Android device without NFC" greatest option. Why can this happen?

**Comment by j-kaltes** [comment]
It is certainly not the same issue as you refer to. That issue was solved long ago with an update of Juggluco.

When you can't scan a sensor with a particular phone, there is something wrong with NFC of that phone.

**Comment by AgushkaNightscout** [comment]
"I changed to a new libre2 sensor and it scans reliably in libre2 app, but juggluco always gets a scan error (18)." - I have exactly the same app behavior (version 8.0.3). Why does FS3 activate without problems but FS2RU does not activate (intercept) with error 18? Do they have such different NFC elements?

**Comment by j-kaltes** [comment]
But you can scan the same Libre 2 sensor with Juggluco on another phone?

**Comment by AgushkaNightscout** [comment]
Yeah, and it's totally confusing. Cubot can't even activate FS2RU with original Abbot's app. I'll try to take it apart and see what's wrong with the NFC module. But why FSL3 can be activated? I can't get my head around)

**Comment by j-kaltes** [comment]
You can run a logging version from https://www.juggluco.nl/Juggluco/download.html#bugreport and e-mail trace.log to me. You can also send logcat.txt:
adb logcat -d >logcat.txt

If it is a US like Libre 2 sensor, the scanning is very complicated.

**Comment by AgushkaNightscout** [comment]
Ok, thank you for advice! Sensor will expire in 5 days, then I will activate (intercept) a new one with the logging version and send you logcat.txt.

**Comment by j-kaltes** [comment]
Why not just scan with you current sensor and send trace.log and logcat.txt?

**Comment by AgushkaNightscout** [comment]
Oh, right! I forgot how cool your app is! When my kid gets home from kindergarten, I'll scan it.

**Comment by gui-dos** [comment]
@j-kaltes:

> If it is a US like Libre 2 sensor, the scanning is very complicated.


Yep: according the attached screenshot that I received, it corresponds to the US 2 newer `2B` type: notice the `A` generation and the `08` regional code:

![311681431-8e853540-f10f-4d16-9d4d-8d7782d4f4a9](https://github.com/j-kaltes/Juggluco/assets/7220550/a649f67b-058a-444f-93fd-6da3f93bccda)

**Comment by AgushkaNightscout** [comment]
Just emailed the logs to you, please take a look at them.

**Comment by j-kaltes** [comment]
I looked at the logs. You scanned two times with two different errors. A scan consists of multiple NFC commands and some of them succeed and others failed. But NFC often fails momentarily because the sensor wasn't placed on the right spot of the phone or so, as was certainly the case with the first error. 
Because I have only one scan, I can't  judge whether scanning never comes past the point of the second scan error.

**Comment by AgushkaNightscout** [comment]
I applied the phone a few times, all the time error 18, please take a look at logs on email.
